esm/index.js

```diff
@@ -1,8 +1,13 @@
+import _pluginJSX from "@babel/plugin-transform-react-jsx";
+// _pluginJSX.default when using native ESM;
+// _pluginJSX when using the version compiled by ascjs.
+const pluginJSX = _pluginJSX.default || _pluginJSX;
```

Comment:
ascjs does this for you already, see the resulting .cjs file

Reply:
unless ... f course, the module doesn't use the `__esModule` convention.

Reply:
With native ESM `_pluginJSX` is something like `{ default: function thePlugin() { ... }, __esModule: true }`, while with ascjs it's `function thePlugin() { ... }`.